experimental/ner_wikiner_speedster/requirements.txt

```diff
@@ -0,0 +1,4 @@
+nebullvm>=0.6
+speedster>=0.0.1
+openvino-dev[onnx]
```

Comment:
I also only added this to try to turn off the telemetry in the tests, but I'm not sure whether it needs to be installed separately or if there's a better way to manage this. (Maybe it's not installed by default anymore for the tests?)